src/qml/About.qml

```diff
@@ -118,18 +118,30 @@ Item {
       Layout.fillWidth: true
       Layout.maximumWidth: parent.width
       Layout.alignment: Qt.AlignCenter
+      Layout.bottomMargin: 10
       horizontalAlignment: Text.AlignHCenter
       font: Theme.tinyFont
-      color: Theme.light
-      opacity: 0.6
+      color: Theme.secondaryTextColor
+      textFormat: Text.RichText
 
       text: {
-        var dataDirs = platformUtilities.appDataDirs();
+        let label = '';
+        let isDesktopPlatform = Qt.platform.os !== "ios" && Qt.platform.os !== "android";
+        let dataDirs = platformUtilities.appDataDirs();
         if (dataDirs.length > 0) {
-          return (dataDirs.length > 1 ? 'QField app directories' : 'QField app directory') + '\n' + dataDirs.join('\n');
+          label = dataDirs.length > 1 ? qsTr('QField app directories') : qsTr('QField app directory');
+          for (let dataDir of dataDirs) {
+            if (isDesktopPlatform) {
+              label += '<br><a href="file://' + dataDir + '">' + dataDir + '</a>';
+            } else {
+              label += '<br>' + dataDir;
+            }
+          }
         }
-        return '';
+        return label;
       }
+
+      onLinkActivated: Qt.openUrlExternally(link)
```

Comment:
All good, just to avoid `Injection of parameters into signal handlers is deprecated` warning we can use this: 
```
onLinkActivated: link => {
  Qt.openUrlExternally(link);
}
```

Reply:
@mohsenD98 , thanks, fixed.